.github/workflows/Ci.yml

```diff
@@ -0,0 +1,103 @@
+# GitHub Actions triggers based on the PR's and each commits on develop and master
+name: "CI: Run tests"
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+    branches: [develop, master]
+jobs:
+  build:
+    defaults:
+      run:
+        shell: bash
+        working-directory: "/opt/carma/"
+
+    # Specify the runner environment
+    runs-on: ubuntu-latest
+
+    # Set up a Docker container for the job
+    container:
+      image: usdotfhwastoldev/autoware.ai:develop
+      env:
+        INIT_ENV: "/home/carma/.base-image/init-env.sh"
+        ROS_2_ENV: "/opt/ros/foxy/setup.bash"
+        TERM: xterm
+      options: "--user root"
+
+    steps:
+    - name: Checkout ${{ github.event.repository.name }}
+      # Check out the repository code
+      uses: actions/checkout@v3.3.0
+      with:
+        fetch-depth: 0
+        path: src/${{ github.event.repository.name }}
+
+    - name: Move source code
+      # Move the source code to the desired location
+      run: mv $GITHUB_WORKSPACE/src /opt/carma/
+
+    - name: Checkout dependencies
+      # Checkout project dependencies
+      run: |
+        source "$INIT_ENV"
+        ./src/${{ github.event.repository.name }}/docker/checkout.bash -r /opt/carma/
+
+    - name: Install external dependencies
+      # Install the multiple object tracking deps
+      run: sudo bash /opt/carma/src/multiple_object_tracking/scripts/install_dependencies.sh
+
+    - name: Build ROS1
+      # Build ROS1 packages
+      # The skipped ROS 1 packages here should match the ROS 1 packages that are skipped during the build in docker/install.sh
+      run: |
+        source "$INIT_ENV"
+        PACKAGES=$(find . -maxdepth 2 -type f -name package.xml | sed 's/\.\///' | cut -d/ -f1) # maxdepth 2 Limits the search of package.xml to current directory and one level of subdirectories beneath it. find will not search beyond two levels deep from the starting point. Without this option, find would search the starting directory and all subdirectories recursively.
+        sed -i 's/colcon build /colcon build --packages-skip novatel_oem7_msgs tracetools tracetools_test /' /home/carma/.ci-image/engineering_tools/code_coverage/make_with_coverage.bash
+        make_with_coverage.bash -m -e /opt/carma/ -o ./coverage_reports/gcov
+
+    - name: Run ROS1 C++ Tests
+      # Run ROS1 C++ tests
+      # The skipped ROS 1 packages here should match the ROS 1 packages that are skipped during the build in docker/install.sh"
+      run: |
+        source "$INIT_ENV"
+        make_with_coverage.bash -t -e /opt/carma/ -o ./coverage_reports/gcov

```

Comment:
I think I made an error in one of my earlier requests. Could we actually add the `sed` command back in here to skip the ROS 1 packages that were skipped during the ROS1 build (`novatel_oem7_msgs`, `tracetools`, and `tracetools_test`)? Without skipping these, I am seeing the unit tests fail for these packages (example below):

```
...
AssertionError: Has this package been built before?
---
Failed   <<< novatel_oem7_msgs [0.00s, exited with code 1]
```